test.js

```diff
@@ -342,7 +342,7 @@ test('flush - time out if configured', async (t) => {
     },
   ];
 
-  await t.throws(client.flush(), 'timeout of 500ms exceeded');
+  await t.throws(client.flush(), `network timeout at: ${host}:${port}/`);
```

Comment:
the error message thrown by `fetch-retry` differs from `axios-retry`. updated test case to match